### DISCUSSION TITLE
Groupbar gap

### DISCUSSION THREAD

**Discussion by matejdro** [hyprwm/Hyprland#6751] [Q&A] [answered]
Hello,

is there a way to disable gap that forms around groupbar buttons?

![image](https://github.com/hyprwm/Hyprland/assets/507922/d05f217c-fc8f-4cf8-9cb3-4503386d5091)

As you can see on above image, there is small border around groupbar, through which my wallpaper is visible. `general:gaps_in` and `general:gaps_out` are already set to 0.

**Comment by syedadeed** [comment, marked as answer]
set group:groupbar:gaps_out = 0

**Reply by matejdro** [reply]
Yup, it seems `0.48.0` finally added this option in. Thanks!

**Comment by tekstryder** [comment]
@matejdro please close this old, answered issue.
